vtex/toolbelt
# Fix SessionManager and account switches

#### What is the purpose of this pull request?
- Use only SessionManager for managing session.
- Remove session related function on conf.
- Create unified function for switching back to previous account
- Remove unnused code
- Refactor `vtex login`
- Do not report `CommandError`.

#### How should this be manually tested?
```
git checkout fix/session-management

yarn watch
```
- Check if `vtex-test login` is working properly - try logging in to an account and check `~/.vtex/session/session.json` and `~/.vtex/session/workspace.json`.
- Check if `vtex-test switch` is working properly
- Try `vtex-test deprecate` an innexistent app to check switching accounts (`vtex-test deprecate basedevmkp.asdf@0.0.1`
- Try `vtex-test undeprecate` an innexistent app to check switching accounts (`vtex-test undeprecate basedevmkp.asdf@0.0.1`
- Try `vtex-test publish` an app with typescript errors to check switching accounts

#### Types of changes

#### Chores checklist
